Solutions/Infoblox Cloud Data Connector/Analytic Rules/HighNumberOfHighThreatLevelDetected.yaml

```diff
@@ -0,0 +1,41 @@
+id: 975f3faf-dcdb-49ae-aafc-9cdf1b2647b8
+name: High Number of High Threat Level Detected
+description: |
+  'This creates an incident in the event a host generates a high number of high threat level queries.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: InfobloxCloudDataConnector
+    dataTypes: 
+      - CommonSecurityLog
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1498
+  - T1565
+query: |
+
+  let timeframe = 1h;
+  let threshold = 200;
+  CommonSecurityLog
+  | where DeviceVendor == "Infoblox" and DeviceProduct == "Data Connector"
+  | where DeviceEventClassID contains "RPZ"
```

Comment:
If possible, use ```has``` instead of ```contains```.  If it is the start or end of a string you can also use ```hasprefix``` or ```hassuffix```.